tests/core/test_library.py

```diff
@@ -419,3 +419,71 @@ def test_core_search_with_handles_legacy_backend(self):
         self.backend.library.search.return_value.get.side_effect = TypeError
         self.core.library.search(query={'any': ['a']}, exact=True)
         # We are just testing that this doesn't fail.
+
+
+@mock.patch('mopidy.core.library.logger')
```

Comment:
Looks like a good place for using pytest's caplog fixture to simplify the logging assertions.